languages/swift/generic/Parse_swift_tree_sitter.ml

```diff
@@ -1750,9 +1765,10 @@ and map_key_path_postfixes (env : env) (x : CST.key_path_postfixes) =
   | `Self tok -> (* "self" *) token env tok
   | `LBRACK_opt_value_arg_rep_COMMA_value_arg_RBRACK (v1, v2, v3) ->
       let lb = (* "[" *) token env v1 in
-      let args = Option.map (map_arguments env) v2 |> Common.optlist_to_list in
-      let rb = (* "]" *) token env v3 in
-      todo env (lb, args, rb)
+      let _args = Option.map (map_arguments env) v2 |> Common.optlist_to_list in
+      let _rb = (* "]" *) token env v3 in
+      (* TODO *)
+      lb
```

Comment:
Is it better to do something like https://github.com/returntocorp/semgrep/pull/8845/files#diff-bc37eef52156d5f5900d5b5e1c601b0bd0c251a3725f7b60b2242093e1e0565dR2666-R2671 instead?

Reply:
I see. This seems fine for now then.